deploy/awstools/awstools.py

```diff
@@ -149,8 +149,19 @@ def get_aws_userid():
     But it seems that by default many accounts do not have permission to run this,
     so instead we get it from instance metadata.
     """
-    res = local("""curl -s http://169.254.169.254/latest/dynamic/instance-identity/document | grep -oP '(?<="accountId" : ")[^"]*(?=")'""", capture=True)
-    return res.stdout.lower()
+    res = None
```

Comment:
In the docstring for this method, change managerinit to awsinit.

Reply:
+1